products-service/handlers/catalogBatchProcess.js

```diff
@@ -0,0 +1,22 @@
+import AWS from 'aws-sdk';
+
+const catalogBatchProcess = async (event) => {
+  console.log(event);
+  const records = event.Records.map(record => record.body);
+  const sns = new AWS.SNS({region: 'us-east-1'});
+  const params = {
+    Message: `New records added: ${JSON.stringify(records)}`,
+    TopicArn: 'arn:aws:sns:us-east-1:489669634691:createProductTopic'
+  };
+
+  try {
+    const result = await sns.publish(params).promise();
```

Comment:
But you should also have code to all items to your tables:
`The lambda function should iterate over all SQS messages and create corresponding products in the products table.`

Reply:
Cool!